pkg/apis/manager/v1alpha3/api.pb.go

```diff
@@ -1032,10 +1032,12 @@ func (m *ValidateAlgorithmSettingsRequest) GetExperiment() *Experiment {
 type ValidateAlgorithmSettingsReply struct {
 }
 
-func (m *ValidateAlgorithmSettingsReply) Reset()                    { *m = ValidateAlgorithmSettingsReply{} }
-func (m *ValidateAlgorithmSettingsReply) String() string            { return proto.CompactTextString(m) }
-func (*ValidateAlgorithmSettingsReply) ProtoMessage()               {}
-func (*ValidateAlgorithmSettingsReply) Descriptor() ([]byte, []int) { return fileDescriptor0, []int{28} }
+func (m *ValidateAlgorithmSettingsReply) Reset()         { *m = ValidateAlgorithmSettingsReply{} }
+func (m *ValidateAlgorithmSettingsReply) String() string { return proto.CompactTextString(m) }
+func (*ValidateAlgorithmSettingsReply) ProtoMessage()    {}
+func (*ValidateAlgorithmSettingsReply) Descriptor() ([]byte, []int) {
+	return fileDescriptor0, []int{28}
+}
```

Comment:
I am not sure that we can edit these lines, since this file is auto generated by this script: https://github.com/kubeflow/katib/blob/master/pkg/apis/manager/v1alpha3/build.sh.
Is it possible to increase lines width for go fmt to avoid this error? In `1.12.5` version we didn't receive fmt error on these lines.

/cc @gaocegege @johnugeorge

Reply:
Oh, you are absolutely right.

+1 to ignore fmt error for these generated files.